app/scenes/Document/components/Document.js

```diff
@@ -174,7 +174,7 @@ class DocumentScene extends React.Component<Props> {
     this.onSave({ publish: true, done: true });
   }
 
-  @keydown(`${meta}+ctrl+h`)
+  @keydown(`${meta}+alt+h`)
```

Comment:
```suggestion
  @keydown("ctrl+alt+h")
```

This binding is different to help text because it maps to "cmd+alt+h" on Mac.